packages/eas-cli/src/commands/workflow/run.ts

```diff
@@ -0,0 +1,115 @@
+import fs from 'node:fs';
+import * as path from 'node:path';
+
+import { getWorkflowRunUrl } from '../../build/utils/url';
+import EasCommand from '../../commandUtils/EasCommand';
+import { EASNonInteractiveFlag } from '../../commandUtils/flags';
+import { WorkflowProjectSourceType } from '../../graphql/generated';
+import { WorkflowRunMutation } from '../../graphql/mutations/WorkflowRunMutation';
+import Log, { link } from '../../log';
+import { getOwnerAccountForProjectIdAsync } from '../../project/projectUtils';
+import { uploadAccountScopedEasJsonAsync } from '../../project/uploadAccountScopedEasJsonAsync';
+import { uploadAccountScopedProjectSourceAsync } from '../../project/uploadAccountScopedProjectSourceAsync';
+
+export default class WorkflowRun extends EasCommand {
+  static override description = 'Run an EAS workflow';
+
+  // TODO(@sjchmiela): Keep command hidden until workflows are live
+  static override hidden = true;
+  static override state = 'beta';
+
+  static override args = [{ name: 'file', description: 'Path to the workflow file to run' }];
+
+  static override flags = {
+    ...EASNonInteractiveFlag,
+  };
+
+  static override contextDefinition = {
+    ...this.ContextOptions.DynamicProjectConfig,
+    ...this.ContextOptions.ProjectDir,
+    ...this.ContextOptions.Vcs,
+    ...this.ContextOptions.LoggedIn,
+  };
+
+  async runAsync(): Promise<void> {
+    const { flags, args } = await this.parse(WorkflowRun);
+
+    Log.warn('Workflows are in beta and subject to breaking changes.');
+
+    const {
+      getDynamicPrivateProjectConfigAsync,
+      loggedIn: { graphqlClient },
+      vcsClient,
+      projectDir,
+    } = await this.getContextAsync(WorkflowRun, {
+      nonInteractive: flags['non-interactive'],
```

Comment:
After giving it some more thought, there isn't an interactive way of using this command, so it's `--non-interactive` by default, right? So I think we can skip the flag altogether?
```suggestion
      nonInteractive: true,
```

Reply:
That's what I thought. However, I figured maybe https://github.com/expo/eas-cli/blob/e45e90a7bad8c612aea5c8186b559b13a19a6a20/packages/eas-cli/src/commands/workflow/run.ts#L44-L47 can be internally interactive? Or https://github.com/expo/eas-cli/blob/e45e90a7bad8c612aea5c8186b559b13a19a6a20/packages/eas-cli/src/commands/workflow/run.ts#L58-L62?